### DISCUSSION TITLE
Any ideas how I got this?

### DISCUSSION THREAD

**Discussion by george-bobby** [Budibase/budibase#1257] [Help] [answered]
![Capture](https://user-images.githubusercontent.com/74462795/110233793-7d380f00-7f4c-11eb-978a-29fb53699bac.PNG)

**Comment by joebudi** [comment, marked as answer]
This is a strange one. If you click 'OK' and try running Budibase, does the error persist?

**Reply by george-bobby** [reply]
Well, I don't know how to use Budibase very well. It is working fine I think(it is doing what I need). But every time I take Budibase, it is showing this error and when I click ok, it will be fine.
